.github/workflows/compile.yml

```diff
@@ -25,6 +25,7 @@ jobs:
         os:
```

Comment:
not related to this change, but this is really a runner, not an os.

It will look better in:

    runs-on: ${{ matrix.runner }}